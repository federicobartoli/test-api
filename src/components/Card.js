import HeartButton from '../components/HeartButton';
//Context

const Card = (props) => {
  return (
    <div onClick={props.handleLike} className="card">
      {props.children}
      <HeartButton active={props.liked} />
    </div>
  );
};

export default Card;
