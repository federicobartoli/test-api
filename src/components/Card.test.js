import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

import Card from './Card';

describe('Card', () => {
  it('renders Card corectly', () => {
    const { container } = render(<Card />);
    expect(container).toMatchSnapshot();
  });
});
