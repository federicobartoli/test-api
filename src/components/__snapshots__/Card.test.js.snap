// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders Card corectly 1`] = `
<div>
  <div
    class="card"
  >
    <button
      class="heart-button "
      data-testid="heart-button"
    >
      <div
        class="heart-flip"
      />
      <span>
        Like
        <span>
          d
        </span>
      </span>
    </button>
  </div>
</div>
`;
